x-pack/legacy/plugins/reporting/server/lib/crypto.ts

```diff
@@ -6,6 +6,10 @@
 
 import nodeCrypto from '@elastic/node-crypto';
 
-export function cryptoFactory(encryptionKey: string | undefined) {
+export function cryptoFactory(encryptionKey?: string) {
+  if (typeof encryptionKey !== 'string') {
```

Comment:
`nodeCrypto` will throw an error if `encryptionKey` is `undefined`: https://github.com/elastic/node-crypto/blob/master/src/crypto.ts#L37

This will type guard the parameter.